src/pages/index.js

```diff
@@ -146,6 +261,13 @@ profileEditButton.addEventListener('click', () => {
 	editProfilePopup.open();
 });
 
+addNewCardButton.addEventListener('click', () => {
+	addCardValidator.resetValidation();
+	addCardValidator.toggleButtonState();
+	addCardPopup.open();
+});
+
+
 profileEditCloseButton.addEventListener('click', () => {
```

Comment:
Please, delete close icon handling from `indexj.s`. Everything is handled by `Popup` from now on